swagger/v1/swagger.yaml

```diff
@@ -668,6 +668,21 @@ paths:
           description: Max depth of recursion. Value must be an integer > 0, or *. Default value is 1.
           required: false
           type: string
+        - name: cr_identifier
+          in: query
+          description: |
+            Identifier of a catalog record. browse only files that have been selected for that record.
```

Comment:
Should it mention here that also internal pk is a valid value for this parameter? It might be useful for service users and why not for endusers as well